manager/manager.go

```diff
@@ -172,7 +188,85 @@ func NewMemoryAlertManager(o *MemoryAlertManagerOptions) AlertManager {
 		inhibitor:          o.Inhibitor,
 		silencer:           o.Silencer,
 		notifier:           o.Notifier,
+		persistenceFile:	o.PersistenceFile,
+	}
+}
+
+// Loads a JSON representation of the memoryManager from a file.
+func (s *memoryAlertManager) loadFromFile() error {
+	s.mu.Lock()
```

Comment:
You probably wouldn't even need any locking here if you called this synchronously before `Run` and starting up anything else from main.